### PR TITLE
etcd-manager/dev: move script interpreter to the first line

### PR DESCRIPTION
**Pull request by zhijianli88**
Signed-off-by: Li Zhijian <lizhijian@cn.fujitsu.com>

### CODE REVIEW

**Comment by dlipovetsky** [comment]
Can you please describe the issue you found that this fixes?

File: etcd-manager/dev/add-symlink.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/bin/bash -e
```
Comment (dlipovetsky) [inline etcd-manager/dev/add-symlink.sh:1]: `-e` will cause the script to exit on error. Why is this flag used here, but not in the other scripts?

Comment (zhijianli88) [reply etcd-manager/dev/add-symlink.sh:1]: Looks -e was here since the initial version, and i just want to fix the interpreter issue first in this pr

**Comment by zhijianli88** [comment]
Only the interpreter appears in the first line makes sense

**Comment by justinsb** [comment]
Thanks @zhijianli88 

/approve
/lgtm
/ok-to-test

**Pull request merged**